File: SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/Cita/CitaDTO.java
package pe.edu.pucp.softinv.model.Cita;

import pe.edu.pucp.softinv.model.Personas.ClienteDTO;
import pe.edu.pucp.softinv.model.Personas.EmpleadoDTO;
import pe.edu.pucp.softinv.model.Servicio.ServicioDTO;

import java.sql.Date;
import java.sql.Time;

public class CitaDTO {
    private Integer id;
    private Time horaIni;
    private Time horaFin;
    private ClienteDTO cliente;
    private ServicioDTO servicio;
    private EmpleadoDTO empleado;
    private Date fecha;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Time getHoraIni() {
        return horaIni;
    }

    public void setHoraIni(Time horaIni) {
        this.horaIni = horaIni;
    }

    public Time getHoraFin() {
        return horaFin;
    }

    public void setHoraFin(Time horaFin) {
        this.horaFin = horaFin;
    }

    public ClienteDTO getCliente() {
        return cliente;
    }

    public void setCliente(ClienteDTO cliente) {
        this.cliente = cliente;
    }

    public ServicioDTO getServicio() {
        return servicio;
    }

    public void setServicio(ServicioDTO servicio) {
        this.servicio = servicio;
    }

    public EmpleadoDTO getEmpleado() {
        return empleado;
    }

    public void setEmpleado(EmpleadoDTO empleado) {
        this.empleado = empleado;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

    public CitaDTO() {
        id=null;
        horaIni=null;
        horaFin=null;
        cliente=null;
        servicio=null;
        empleado=null;
        fecha=null;
    }

    public CitaDTO(Integer id,Time horaIni,Time horaFin,ClienteDTO cliente,ServicioDTO servicio,EmpleadoDTO empleado,
                   Date fecha) {
        this.id = id;
        this.horaIni = horaIni;
        this.horaFin = horaFin;
        this.cliente = cliente;
        this.servicio = servicio;
        this.empleado = empleado;
        this.fecha = fecha;
    }
}

File: C#/SoftInvModel/Cita/CitaDTO.cs
using System;

namespace softinv.model
{
    public class CitaDTO
    {
        private int? id;
        private DateTime? horaIni;
        private DateTime? horaFin;
        private ClienteDTO cliente;
        private ServicioDTO servicio;
        private EmpleadoDTO empleado;
        private DateTime? fecha;

        public CitaDTO()
        {
            this.Id = null;
            this.HoraIni = null;
            this.HoraFin = null;
            this.Cliente = null;
            this.Servicio = null;
            this.Empleado = null;
            this.Fecha = null;
        }

        public CitaDTO(int id, DateTime horaIni, DateTime horaFin, ClienteDTO cliente, ServicioDTO servicio, EmpleadoDTO empleado,
                       DateTime fecha)
        {
            this.Id = id;
            this.HoraIni = horaIni;
            this.HoraFin = horaFin;
            this.Cliente = cliente;
            this.Servicio = servicio;
            this.Empleado = empleado;
            this.Fecha = fecha;
        }

        public int? Id { get => id; set => id = value; }
        public DateTime? HoraIni { get => horaIni; set => horaIni = value; }
        public DateTime? HoraFin { get => horaFin; set => horaFin = value; }
        public ClienteDTO Cliente { get => cliente; set => cliente = value; }
        public ServicioDTO Servicio { get => servicio; set => servicio = value; }
        public EmpleadoDTO Empleado { get => empleado; set => empleado = value; }
        public DateTime? Fecha { get => fecha; set => fecha = value; }
    }
}


